spec/controllers/new_administrateur/sources_particulier_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe NewAdministrateur::SourcesParticulierController, type: :controller do
+  let(:admin) { create(:administrateur) }
+
+  before { sign_in(admin.user) }
+
+  describe "#show" do
+    let(:procedure) { create(:procedure, administrateur: admin, api_particulier_scopes: ['cnaf_enfants'], api_particulier_sources: { cnaf: { enfants: ['noms_prenoms'] } }) }
+
+    render_views
+
+    subject { get :show, params: { procedure_id: procedure.id } }
+
+    it { expect(subject.body).to include(I18n.t('api_particulier.providers.cnaf.scopes.enfants.date_de_naissance')) }
+    it { expect(subject.body).to have_selector("input#procedure_cnaf_enfants_noms_prenoms[checked=checked]") }
+
+    it { expect(subject.body).to have_selector("input#procedure_cnaf_enfants_date_de_naissance") }
+    it { expect(subject.body).not_to have_selector("input#procedure_cnaf_enfants_date_de_naissance[checked=checked]") }
```

Comment:
Le setup de chaque spec est assez lent : en ce moment un `create(:procedure)` prend presque 1s, tellement y'a d'objets associés à créer en base.

Un jour on améliorera ça (probablement en définissant des procédures statiques dans les `fixtures.yml` (si quelqu'un veut se pencher là dessus, please do !)), mais pour l'instant ça veut dire que limiter le nombre de `it` est sans doute une bonne idée.

```suggestion
    it 'renders the sources form' do
      expect(subject.body).to include(I18n.t('api_particulier.providers.cnaf.scopes.enfants.date_de_naissance'))
      expect(subject.body).to have_selector("input#procedure_cnaf_enfants_noms_prenoms[checked=checked]")
      
      expect(subject.body).to have_selector("input#procedure_cnaf_enfants_date_de_naissance")
      expect(subject.body).not_to have_selector("input#procedure_cnaf_enfants_date_de_naissance[checked=checked]")
    end
```